config.h.meson

```diff
@@ -136,7 +136,7 @@
 #mesondefine URANDOM
 
 /* Version number of package */
-#mesondefine VERSION
+#define VERSION "@VERSION@"
```

Comment:
I think you just want to use `conf_data.set_quoted` and leave this a `#mesondefine`.

Reply:
Ah indeed, I completely missed the `set_quoted` variant. Fixed locally, thanks!